CONTRIBUTING.md

```diff
@@ -99,7 +88,6 @@ be considered for inclusion in the code base and history of this repository.
 * Make commits of logical units.
 * Check for unnecessary whitespace with `git diff --check` before committing.
 * Make sure your commit messages are [well formed](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
-* If you created an issue, you can close it by including "Closes #issue" in your commit message. See [Github's blog post for more details](https://github.com/blog/1386-closing-issues-via-commit-messages)
```

Comment:
we do not want to close issues when a pr is closed now that we're using a project board and release cadence.